Dockerfiles/ccp-openshift-slave/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM registry.centos.org/openshift/jenkins-slave-base-centos7
+
+RUN yum install -y PyYAML && \
+    yum install -y docker && \
+    yum install -y mailx && \
+    yum install -y epel-release && \
+    yum install -y npm && \
+    yum clean all && \
+    npm install -g dockerfile_lint
```

Comment:
how are we planning to manage the plugins, we need extra then the default ones?

Reply:
I'm not sure if I understand the question. Which plugins are we concerned about here?

Reply:
As discussed, if we want to install plugins in Jenkins server, we'll have to do something like what I've done [here](https://github.com/dharmit/dockerfiles/tree/master/ccp-jenkins). A separate container image needs to be created that reads a plugins.txt. This plugins.txt needs to have the plugins we want to install.

Reply:
```
yum -y install -y PyYAML docker mailx epel-release && \
yum -y isntall npm &&
yum clean all && \
[..]
```
should do as well - we can club the RPM package names together in on `yum install -y` command